utils/sub8_msgs/srv/BMatrix.srv

```diff
@@ -0,0 +1,2 @@
+---
+float64[] B
```

Comment:
The thruster mapper internally changes B when a thruster fails - perhaps we should change this in light of both the TGEN and the mapper using the same matrix.

Ignore for now and let's talk about it on Weds